src/ProblemDetailsResponseFactory.php

```diff
@@ -16,7 +16,6 @@
 use Psr\Http\Message\ServerRequestInterface;
 use Spatie\ArrayToXml\ArrayToXml;
 use Throwable;
-
```

Comment:
We use an empty line between class imports and function imports, and again between function and constant imports. Please re-instate those breaks.

Reply:
Sorry, PHPStorm autoformat. :) Reverted the formatting.